src/libs/ReportUtils.js

```diff
@@ -531,6 +545,26 @@ function isPolicyExpenseChatAdmin(report, policies) {
     return policyRole === CONST.POLICY.ROLE.ADMIN;
 }
 
+/**
+ * Returns true if report is a DM/Group DM chat.
+ *
+ * @param {Object} report
+ * @returns {Boolean}
+ */
+function isDM(report) {
+    return !getChatType(report);
+}
+
+/**
+ * If the report is a thread and has a chat type set, it is a workspace chat.
```

Comment:
Are there no other types of threads that have a `chatType`? Running into a problem because a thread with `chatType == groupChat` will return `true` for a "workspace thread".

What is the actual `chatType` for a "workspace thread"? 😅

Reply:
Sorry, I figured out the answer to this. Nothing to see here.

`chatType` might not mean workspace related room these days. But definitely did when we added this!